pep-0674.rst

```diff
@@ -231,58 +225,72 @@ losing support with older Python versions. The project provides a header
 file which provides ``Py_SET_REFCNT()``, ``Py_SET_TYPE()`` and
 ``Py_SET_SIZE()`` functions to Python 3.8 and older.
 
-PyTuple_GET_ITEM() and PyList_GET_ITEM()
-----------------------------------------
-
-The ``PyTuple_GET_ITEM()`` and ``PyList_GET_ITEM()`` macros are left
-unchanged.
-
-The code pattern ``&PyTuple_GET_ITEM(tuple, 0)`` and
-``&PyList_GET_ITEM(list, 0)`` is still commonly used to get access to
-the inner ``PyObject**`` array.
-
-Changing these macros is out of the scope of this PEP.
 
 
 Backwards Compatibility
 =======================
 
 The proposed C API changes are backward incompatible on purpose.
 
-At December 1, 2021, a code search on the PyPI top 5000 projects (4760
+At January 26, 2022, a code search on the PyPI top 5000 projects (4762
 projects in practice, others don't have a source archive) found that
-`only 14 projects are affected
-<https://bugs.python.org/issue45476#msg407456>`_ (0.3%):
+only 13 projects are affected (0.3%):
```

Comment:
This duplicates the above -- I'd merge them. I'm not sure the line around the 238 projects without sdists is needed.